libraries/Servo/src/Servo.cpp

```diff
@@ -68,11 +68,11 @@ uint8_t Servo::attach(int pin, uint16_t minUs, uint16_t maxUs)
     _attached = true;
   }
 
-  // keep the min and max within 200-3000 us, these are extreme
+  // keep the min and max within 500-2500 us, these are extreme
```

Comment:
I disagree with this change.  There are servos out there that require the wider range, this is meant to keep outrageous values from being used, not to maintain "safe" values for your servos.